docs/challenge/updates.md

```diff
@@ -1,58 +1,90 @@
+1. We updated the [submission guildeline](./submission.md) and added a [sample submission Dockerfile](https://github.com/StanfordVL/BEHAVIOR-1K/blob/main/OmniGibson/docker/submission.Dockerfile) for reference.  
+2. We are excited to announce that [NVIDIA](https://www.nvidia.com/en-us/) will be sponsoring our challenge!  
+   The updated prize pool is as follows:
+    - **First Place:** $1000 + GeForce 5080  
+    - **Second Place:** $500 + (Jetson Orin Nano Super *or* $1000 Brev Credits)  
+    - **Third Place:** $300 + $500 Brev Credits  
 
+---
 
-
-### 09/19/2025
+### 09/28/2025
 
 **Challenge rule clarifications:**
 
-1. BDDL task definitions are allowed in the standard track. These task definitions are fixed and will be the same during evaluation.
+1. No formal registration is required to participate in the challenge.  
+   Feel free to submit your results directly if you have one!
 
-2. Collecting more data by yourself (with teleoperation, RL, scripted policies, etc.) is allowed for standard track. Do note, however, that you are not allowed to collect data on the evaluation instances, those are meant to test the generalization capability of the submitted policy. 
+**Bug fixes:**
 
-3. There are no restrictions on the form of the policy for both tracks. It could be IL, RL, TAMP, etc. Components like SLAM, querying LLMs, are also allowed. 
+1. Fixed multi-worker sharding and action chunk indexing in `BehaviorLeRobotDataset` under chunk streaming mode.  
+2. Fixed incorrect robot start pose in the evaluation script.  
+3. Provided improved baseline checkpoints.  
+   Please refer to [baselines.md](./baselines.md) for details.
 
-3. At this point, the success score (Q) is the only metric used for ranking submissions. If two submissions have the same score, secondary metrics will be used to break ties.
+All fixes have been pushed to the `main` branch.
 
-4. The timeout for each evaluation is set to be 2 * mean task completion time of the 200 human demos, thus it varies across tasks. 
+**New features:**
 
-5. Besides the 200 human collected demos, we provided 20 additional configuration instances for each task. You should use the first 10 instances to get the evaluation results (see [evaluation.md](./evaluation.md#evaluation-protocol-and-logistics)); the latter 10, however, are not used for evaluation. Feel free to use those as a validation set before running your policy against the first 10 evaluation instances.
+1. Added several new CLI arguments for evaluation, including  
+   `testing_on_train_instances`, `early_stopping`, and `partial_scene_load`.  
+   See [base_config.yaml](https://github.com/StanfordVL/BEHAVIOR-1K/blob/main/OmniGibson/omnigibson/learning/configs/base_config.yaml) for more details.
```

Comment:
This sounds possibly interesting (partial scene load etc) - do we perhaps want to talk a bit more about this on the eval page and whatnot?